app/Http/Livewire/EditLink.php

```diff
@@ -12,11 +12,14 @@ class EditLink extends Component
     public $slug;
     public $is_enabled;
 
-    protected $rules = [
-        'url' => 'required|url|max:255',
-        'slug' => 'required|alpha_dash|min:3|max:100',
-        'is_enabled' => 'required|boolean',
-    ];
+    public function rules()
+    {
+        return [
+		'url' => 'required|url|max:255',
+		'slug' => 'required|alpha_dash|min:3|max:100|unique:links,slug,'.$this->link_id,
+		'is_enabled' => 'required|boolean',
+	    ];
+    }
```

Comment:
Can you leave this as a protected property and make sure it is spaced correctly.

Also, we can do the string concatenation like this:
`'slug' => "required|alpha_dash|min:3|max:100|unique:links,slug,{$this->link_id}",`

Reply:
Hi @davidgrzyb. I have updated the string concatenation. Converting the rules to a property, concatenating with `$this->link_id` throws an error the error below. 
![image](https://user-images.githubusercontent.com/1807304/105961335-6f5eb680-608f-11eb-8974-1f3d437b5f81.png)

The alternative would be to leave it as a property and add the following line within the `mount()` method : 
```PHP
$this->rules['slug'] = "required|alpha_dash|min:3|max:100|unique:links,slug,{$this->link_id}";
```
I had thought converting the property to a method which allows string concatenations is a more cleaner solution

Reply:
I have pushed a commit with a work around that retains the rules property. This helps maintain the code as written during the video. Kindly review and advise.